app/models/calendar.rb

```diff
@@ -45,6 +45,14 @@ def show_bunting?
     divisions.any?(&:show_bunting?)
   end
 
+  def bunting_styles
+    if [1, 12].include? Date.today.month
```

Comment:
Yep, and the whole of December - but historically I think the only bank holidays in those 2 months are around Christmas and New Year.